tika-eval/src/main/java/org/apache/tika/eval/EvalFilePaths.java

```diff
@@ -81,9 +82,8 @@ public boolean equals(Object o) {
 
         if (sourceFileLength != that.sourceFileLength) return false;
         if (extractFileLength != that.extractFileLength) return false;
-        if (relativeSourceFilePath != null ? !relativeSourceFilePath.equals(that.relativeSourceFilePath) : that.relativeSourceFilePath != null)
-            return false;
-        return extractFile != null ? extractFile.equals(that.extractFile) : that.extractFile == null;
+        if (!Objects.equals(relativeSourceFilePath, that.relativeSourceFilePath)) return false;
```

Comment:
In general I think Tika uses a "always braces" style, so this would be better as:

```java
if (!blah) {
    return false;
} else {
    return Objects.equals(blah);
}
```

Reply:
Sure. And this is also the style I prefer. I made the `if` clause like this cause I want to have the same style as line 83, 84. Will modify all these 3 lines.